src/api/render.js

```diff
@@ -0,0 +1,9 @@
+import registry from '../global/registry';
```

Comment:
This is the meat of the PR. This just forces the render lifecycle on the element.